xarray/backends/h5netcdf_.py

```diff
@@ -140,6 +140,7 @@ def open(
         invalid_netcdf=None,
         phony_dims=None,
         decode_vlen_strings=True,
+        driver=None,
```

Comment:
We might want to enable `driver_kwds`, too. See https://docs.h5py.org/en/stable/high/file.html#file-drivers. With that users would be able to correctly specify eg. authentication.

Reply:
```suggestion
        driver=None,
        driver_kwds=None,
```